src/domain/Lists/DraggableList/style.ts

```diff
@@ -22,8 +22,9 @@ const StyledListItem = styled.div<IListItemProps>`
 `
 
 const StyledListArea = styled.div`
-  display: grid;
-  row-gap: ${Variables.Spacing.sXSmall}px;
+  > * {
```

Comment:
> *:not(:last-child) {

}
If this works, it could be equal to margins on row gap I guess

Reply:
I considered doing that, thought I have a feeling the library might not like it. Their example didn't remove the last child's margin but I'll give it a try.

Reply:
Worse case we'll just have to make `StyledListArea` do `margin-bottom: -${Variables.Spacing.sXSmall}px;` if it doesnt work.